### PR TITLE
Valkyrie Map, New Areas

### PR DESCRIPTION
**Pull request by Vasily2013**
## About The Pull Request

This PR adds in multiple new areas for ship based structure that is used in the project, including frame and deck numbered hallways and maint areas.

It also adds the meme ship Valkyrie

## Why It's Good For The Game

Because knowing where you're going is amazingly good, especially on crew monitors and air/power alarms.

## Changelog
:cl:
add: various areas
add: Valkyrie
/:cl:

valk1
valk2
Valk3

### CODE REVIEW

**Comment by Kmc2000** [comment]
bsg13 NOW.

**Comment by Wilchenx** [comment]
pics please

File: nsv13/game/areas.dm [hunk 1]
```diff
@@ -27,6 +27,204 @@
...
+/area/hallway/nsv/deck3/frame4/port
+	name = "Deck 3 Frame 4 Port Hallway"
+	icon_state = "hallP"
+
+/area/hallway/nsv/deck3/frame4/starboard
+	name = "Deck 3 Frame 4 Starboard Hallway"
+	icon_state = "hallS"
+
+/area/hallway/nsv/deck1/hallway
+	name = "Deck 1 Primary Hallway"
```
Comment (Karmic-Skink) [inline nsv13/game/areas.dm]: Inconsistent naming procedure using letters for some and numbers for others.

File: nsv13/game/areas.dm [hunk 2]
```diff
@@ -27,6 +27,204 @@
...
+/area/crew_quarters/nsv/observation
+	name = "Observation Lounge"
+	icon_state = "Sleep"
+
+/area/crew_quarters/nsv/officerquarters
+	name = "Bridge Officer's Quarters"
+	icon_state = "Sleep"
+
+/area/nsv/hanger/deck3/starboard
+	name = "Deck Three Starboard Hanger Bay"
```
Comment (Karmic-Skink) [inline nsv13/game/areas.dm]: Inconsistent naming procedure using letters for some and numbers for others.

Comment (Vasily2013) [reply nsv13/game/areas.dm]: Resolved by d30a32d

**Comment by Karmic-Skink** [comment]
Potty Mouth.

**Comment by Vasily2013** [comment]
You know I jest, Karmic. Seriously, thanks for catching that.

File: config/maps.txt [hunk 3]
```diff
@@ -30,3 +30,6 @@ endmap
 
 map multiz_debug
 endmap
+
+map Valkyrie
```
Comment (francinum) [inline config/maps.txt]: Move this above runtime.

Comment (Vasily2013) [reply config/maps.txt]: Fixed in c78af28

**Comment by Vasily2013** [comment]
So, merge when?

**Pull request merged**